styles/gauge.json

```diff
@@ -0,0 +1,564 @@
+					"gauge": "914"
+				}
+			}],
+			"caption": "914mm"
+		},
+		{
+			"minzoom": 10,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+					"railway": "rail",
+					"gauge": "950"
+				}
+			}],
+			"caption": "950mm"
+		},
+		{
+			"minzoom": 2,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+					"railway": "rail",
+					"gauge": "1000",
+					"usage": "main"
+				}
+			}],
+			"caption": "1000mm"
+		},
+		{
+			"minzoom": 10,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+					"railway": "rail",
+					"gauge": "1009"
+				}
+			}],
+			"caption": "1009mm"
+		},
+		{
+			"minzoom": 10,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+					"railway": "rail",
+					"gauge": "1050"
+				}
+			}],
+			"caption": "1050mm"
+		},
+		{
+			"minzoom": 2,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+					"railway": "rail",
+					"gauge": "1067",
+					"usage": "main"
+				}
+			}],
+			"caption": "1067mm"
+		},
+		{
+			"minzoom": 10,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+					"railway": "rail",
+					"gauge": "1100"
+				}
+			}],
+			"caption": "1100mm"
+		},
+		{
+			"minzoom": 10,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+					"railway": "rail",
+					"gauge": "1372"
+				}
+			}],
+			"caption": "1372mm"
+		},
+		{
+			"minzoom": 10,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+					"railway": "rail",
+					"gauge": "1422"
+				}
+			}],
+			"caption": "1422mm"
+		},
+		{
+			"minzoom": 10,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+					"railway": "rail",
+					"gauge": "1432"
+				}
+			}],
+			"caption": "1432mm"
+		},
+		{
+			"minzoom": 2,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+					"railway": "rail",
+					"gauge": "1435",
+					"usage": "main"
+				}
+			}],
+			"caption": "1435mm"
+		},
+		{
+			"minzoom": 10,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+					"railway": "rail",
+					"gauge": "1445"
+				}
+			}],
+			"caption": "1445mm"
+		},
+		{
+			"minzoom": 10,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+					"railway": "rail",
+					"gauge": "1450"
+				}
+			}],
+			"caption": "1450mm"
+		},
+		{
+			"minzoom": 10,
+			"features": [{
+				"type": "LineString",
+				"coordinates": [[10, 50], [80, 50]],
+				"properties": {
+		      			"railway": "rail",
```

Comment:
This line has a white space error, there are a few spaces between the tabs. Please remove them.

Reply:
done